waiter/src/waiter/utils.clj

```diff
@@ -389,6 +389,22 @@
   ([slots-in-use slots-available slots-offered outstanding]
    (- outstanding (+ slots-in-use slots-available slots-offered))))
 
+(defn requires-help?
```

Comment:
Can we make `requires-help` have the same tense as `compute-help-required?`

Reply:
`release-unneeded-work-stealing-offers!` does not take into account `requires-help?`.  If we reject offers based upon having available slots, should we not also release unneeded offers based upon the same criteria?

Reply:
Both `help-required?` and `compute-help-required` deal with work-stealing and will never be used outside of that context.  Do we not like `waiter.state` having to depend on `waiter.work-stealing`?
  `utils` is for general use functions.  It's not a place we put things because two namespaces require it.

Reply:
Okay, moved to `waiter.work-stealing`